### DISCUSSION TITLE
DBSchema what are the Status Codes?

### DISCUSSION THREAD

**Discussion by daniel-ls-howard** [nextgenhealthcare/connect#5984] [Q&A] [answered]
I have a channel which shows '5' Queued but when I search in the dashboard I find no records. I'm trying to find where this value is to see if I can clean it up. It's been there for ages and I don't know why.

Looking at the code I see a query:
		SELECT COUNT(*)
		FROM D_MM${localChannelId}
		WHERE ID = ? AND STATUS = ? AND SERVER_ID = ?

What are the STATUS Values to try? 

I found T,S,R and would there be Q?   What do the T, S, R stand for?

thanks

**Comment by pacmano1** [comment, marked as answer]
I think transformed, sent, received and queued.

Thanks @jonbartels, from your link below:

```JS
RECEIVED('R'), FILTERED('F'), TRANSFORMED('T'), SENT('S'), QUEUED('Q'), ERROR('E'), PENDING('P');
```

**Reply by daniel-ls-howard** [reply]
OK and Q for Queued and I found "F". Not sure about that one.

**Comment by jonbartels** [comment]
https://github.com/nextgenhealthcare/connect/blob/c1bbf981b396279984124adbd50ddb4851b2bc76/donkey/src/main/java/com/mirth/connect/donkey/model/message/Status.java#L17
